eman2/tests/test_protocols_eman.py

```diff
@@ -34,8 +34,10 @@
 from pwem.objects.data import Pointer
 from pyworkflow.utils import magentaStr
 
+from ..constants import TOMO_NEEDED_MSG
 from .. import Plugin
```

Comment:
unused import

Reply:
Fixed in https://github.com/scipion-em/scipion-em-eman2/pull/64/commits/3b805869019181e3e364f9837a73ef99f09c12dd